docs/examples.rst

```diff
@@ -338,10 +338,30 @@ directory::
     url: https://github.com/teemtee/tmt.git
     Test case 'TC#0603489' successfully exported to nitrate.
 
-Use the ``--bugzilla`` option together with ``--nitrate`` to link
-bugs marked as ``verifies`` in the :ref:`/spec/core/link`
-attribute with the corresponding Nitrate test case.
-
+    $ tmt test export --how polarion --project-id RHIVOS --create .
+    Test case 'RHIVOS-343' created.
+    summary: tmt/tests/integration/data/polarion/existing_testcase - This is case what already exists inside polarion
+    script: https://github.com/teemtee/tmt.git
+    tags: integration fmf-export
+    status: approved
+    Append the Polarion test case link.
+    implements: https://polarion.example/polarion/#/project/RHIVOS/workitem?id=RHIVOS-343
+    Test case 'This is case what already exists inside polarion' successfully exported to Polarion.
+
+
+Use the ``--bugzilla`` option together with ``--how nitrate`` or
+``--how polarion`` to link bugs marked as ``relates``
```

Comment:
yeah, documentation was wrong previously, all links to bugzilla are imported as relates, so sticking with that

Reply:
I think there is misunderstanding here. "import" import bugs as relates (from tcms case to tmt file) because in tcms you can't tell if it is 'bug which is verified by the test' or 'bug got hit by test run' or something else.

However "export" part is strict - bug has to be in "verifies" relation for test export to create external link in the bugzilla.

So  for 'tmt test export' documentation was correct and should stay as is (saying 'verifies')